pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md

```diff
@@ -17,22 +17,13 @@ facebookenable: true
 ---------------------
 This new channel was added in Mautic 1.4.0. It allows Mautic to send text messages from campaigns.
 
-## Configure Text Messages
-
-Before you start to send text messages from your Mautic, it needs to be connected to the service which can send them. The first and default implemented service is [Twilio][twilio]. In order to configure the text messages correctly, follow these steps:
-
-1. Create an account at Twilio.com.
-2. In Mautic, go to *Settings* (cog icon) > *Plugins*.
-3. Open *Twilio* plugin and activate it.
-4. Copy the *Account SID* from Twilio account and paste it to *Account SID* field in the Twilio plugin configuration.
-5. Unlock and copy the *Auth Token* and paste it to *Auth Token* field in the Twilio plugin configuration.
-6. Go to *Products* > *Phone Numbers* in Twilio, copy the number and paste it to the *Sending Phone Number* field in Mautic.
-7. Select the *Text Message Enabled?* switch to *Yes* and save the Mautic configuration.
 
 ## Create a new Text Message
 
 A Text Message can be created/modified only via Campaign Builder.
 
+Before you start to send text messages from your Mautic you need to configure the text messages correctly see [Twilio Plugin](./../plugins/Twilio), Then follow these steps:
+
```

Comment:
To link to an existing page just grab everything after /en and put that in the link, ensuring that you use link macros instead of in-line links.

In this case we would add the link of /plugins/twilio

Reply:
@RCheesley thanks it works